app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -1,121 +1,434 @@
+<!--   -->>read3/1
+<!--   -->CCCTCAAGCCCGATCGAACTTTCCTACTGCCCGACCTCAGCGCGGAGCATTACCGAATCGTCGTCAACAATCTCCCCGATGGCTTCTATGTGAACTCCAT
+<!--   -->>read3/2
+<!--   -->CAAAATGGAGTGGGCGCTACTGCTCCGTGAGCAGGGTCAGTTGGAGGGATTGGGGCGTGCGCTCGGGGATGCTAATGGCGGTGCCCCTGGATTCCTGAGA
+        </pre>
+        <p>
+          This command will interleave the given FASTQ files and output a FASTA stream,
+          in which the paired-ends alternate each other.
+        </p>
+      </div>
+    </div>
 
-    if line.startswith('>'):
-        header = line.split('_')[0]
-        if header != prevheader:
-            prevheader = header
-            sys.stdout.write('{}\n'.format(header))
-        else:
-            skipped += 1
-            sys.stderr.write('{}: skipped {}\n'.format(skipped, header.rstrip()))
-    else:
-        sys.stdout.write(line)</pre>
-        <p>
-          <b>NOTE</b>: The file <code>singles_proteins.fasta.faa</code> may contain two mates coming from the same clone, but that are not consecutive in the file. Since we know that the protein fragments from both mates come from the same organism, the processing could be improved if we also join the peptides from the protein fragments of the same clone. This is not yet covered by the python script. This could be done by sorting the results after the next step (after <code>unipept pept2lca</code>).
-    </p>
-        <p>
-          Then we compute the LCA for all individual tryptic peptides <i>(timing: 37 mins 17 secs)</i>.
-    </p>
-        <pre>
-<b>$</b> unipept pept2lca < peptides.fst > peptides.lca.fst</pre>
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Gene prediction</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          To convert the metagenomics reads into a metaproteomics problem,
+          we have to perform some kind of gene prediction.
+          While any gene prediction tool can be used, UMGAP was tested with FragGeneScan (2010) and FragGeneScan++,
+          our in-house improved version of FragGeneScanPlus (2015), a predictor using on Hiddem Markov Models.
+          On the other hand you can choose not to bother with prediction
+          and just translate all six reading frames completely with a builtin tool.
+        </p>
         <p>
-          Instead of deriving the biodiversity distribution from the individual peptides, we can aggregate the identifications of all peptides coming from the same DNA read into a consensus identification. To do so, we use a special version of the lowest common ancestor algorithm (LCA*) that takes the most specific node in the taxonomic tree if both nodes are on the same lineage <i>(timing: 6 mins 36 secs)</i>.
-    </p>
+          This step in the pipeline limits our tool to the coding regions of the given DNA reads.
+          This caps the sensitivity of our predictions,
```

Comment:
When writing, always use as few ambiguous this'es as possible. A good practice is to create "overlapping sentences" where a new sentence begins with something that was introduced in the previous one. This makes the text easy to follow. In the text, instead of "this step" you can write gene prediction step and the second this could be replace by something like "By limiting us to coding regions, the sensitivity of our predictions is capped..."

_[Original comment](https://github.ugent.be/unipept/unipept/pull/642#discussion_r16550) by @bmesuere on Mon Aug 20 2018 at 13:47._